### PR TITLE
[Install] Some updates

### PR DESCRIPTION
**Pull request by phansys**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT
| Doc PR        |

* SemVer compliant via ```branch-alias```
* Updated install docs using composer

### CODE REVIEW

**Comment by Ocramius** [comment]
@phansys can you remove and gitignore the `composer.lock` please?

**Comment by phansys** [comment]
@Ocramius, done.

**Comment by Ocramius** [comment]
@phansys thanks! Sorry, but I have to ask you also for a rebase :-(

**Comment by phansys** [comment]
No problem at all @Ocramius, rebased ;)

**Comment by Ocramius** [comment]
:+1:

**Comment by phansys** [comment]
Thank you @Ocramius!

File: composer.json [hunk 1]
```diff
@@ -10,11 +10,16 @@
         {"name": "Steve Müller", "email": "st.mueller@dzh-online.de"}
     ],
     "require": {
-        "php": ">=5.3"
+        "php": ">=5.3",
+        "squizlabs/php_codesniffer": "~1.0"
```
Comment (gianarb) [inline composer.json]: Hi! Sorry but why ~1.0?! Now php_codesniffer has 2.0 release

Comment (TomasVotruba) [inline composer.json]: @gianarb It was just copied from line bellow: https://github.com/doctrine/coding-standard/pull/4/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780L16

For bumping to PHP_CodeSniffer 2.0 I'd recommend standalone PR. There are many BC breaks.

Comment (gianarb) [inline composer.json]: Ok I noticed this change because in doctrine/DoctrineModule composer was broken :)

Comment (TomasVotruba) [inline composer.json]: @gianarb  You mean this PR has broken something? Could you send link?

Comment (gianarb) [inline composer.json]: https://github.com/doctrine/DoctrineModule/commit/4b62a2ca9480be2ddd32a0ab62f87d91776a4021

**Comment by TomasVotruba** [comment]
@gianarb I see, thanks.

File: composer.json (hunk 1 above)
Comment (TomasVotruba) [inline composer.json]: @gianarb I see, thanks.

**Pull request merged**